README.md

```diff
@@ -44,6 +44,16 @@ docker run -dit --name debugger-tcpdump -v ~/.aws/:/root/.aws/ -e BUCKET={bucket
           "protocol": "tcp",
           "containerPort": 80
         }
+      ],  
+      "environment": [
+        {
+          "name": "BUCKET",
+          "value": "databackupbucketelblogs"
```

Comment:
Could you change this to a more generic name to indicate it needs filling in?

Reply:
updated bucket name